bind.yaml

```diff
@@ -64,27 +83,14 @@ pipeline:
 
   - runs: |
       # removed -g named -o root
-      install -d -m0770 ${{targets.destdir}}/var/bind \
-        ${{targets.destdir}}/var/bind/sec \
-        ${{targets.destdir}}/var/bind/dyn
-
-      # removed -g named -o root
-      install -d -m0750 ${{targets.destdir}}/etc/bind \
-        ${{targets.destdir}}/var/bind/pri
-
-      make -j1 DESTDIR=${{targets.destdir}} install
```

Comment:
removing this as we already have `autoconf/make-install` pipeline doing the same thing